root/report/index.tt

```diff
@@ -108,6 +108,7 @@
             <li><a href="[% c.uri_for_action('/report/show', 'ReleasesWithNoMediums') %]">[% l('Releases with no mediums') %]</a></li>
             <li><a href="[% c.uri_for_action('/report/show', 'ReleasesWithoutVACredit') %]">[% l('Releases not credited to "Various Artists" but linked to VA') %]</a></li>
             <li><a href="[% c.uri_for_action('/report/show', 'ReleasesWithoutVALink') %]">[% l('Releases credited to "Various Artists" but not linked to VA') %]</a></li>
+            <li><a href="[% c.uri_for_action('/report/show', 'ReleasesWithoutDiscIds') %]">[% l('Releases without Disc IDs') %]</a></li>
```

Comment:
The URL is fine, but the title is a bit confusing. It is legitimate to have (e.g. Vinyl) releases without Disc IDs. The title should explain the weirdness of reported releases. Replacing _without_ by _missing_ will probably do.